docs/content/stable/reference/configuration/yugabyted.md

```diff
@@ -32,6 +30,28 @@ Running YugabyteDB on macOS requires additional settings. For more information,
 
 {{% /note %}}
 
+## Installation
+
+The yugabyted executable file is packaged with YugabytDB and located in the YugabyteDB home `bin` directory.
+
+If you want to use [backup](#backup) and [restore](#restore), you also need to install YB Controller:
+
+1. Download the [YB Controller release](https://downloads.yugabyte.com/ybc/2.1.0.0-b9/ybc-2.1.0.0-b9-linux-x86_64.tar.gz).
+1. Extract the `ybc-2.0.0.0-b19-linux-x86_64.tar.gz` file into the `yugabytedb/ybc` folder as follows:
+
+    ```sh
+    cd yugabyte-{{< yb-version version="stable" >}}
+    mkdir ybc | tar -xvf share/ybc-2.0.0.0-b19-linux-x86_64.tar.gz -C ybc --strip-components=1
```

Comment:
I don't think users will download the binary into the `share` folder. 

So we will need to change the following.

1. Download the YB controller release into the `YugabyteDB/share` folder.

```sh
cd yugabyte-{{< yb-version version="stable" >}}/share
wget https://downloads.yugabyte.com/ybc/2.1.0.0-b9/ybc-2.1.0.0-b9-linux-x86_64.tar.gz
```